amplify/team-provider-info.json

```diff
@@ -15,25 +15,29 @@
       "function": {
         "expressServer": {
           "deploymentBucketName": "amplify-stockapp-dev-151619-deployment",
-          "s3Key": "amplify-builds/expressServer-746764634d6933773954-build.zip"
+          "s3Key": "amplify-builds/expressServer-753174514c376f647362-build.zip"
         },
         "twitterScraping": {
           "secretsPathAmplifyAppId": "d2l3aeky8jssr5",
           "deploymentBucketName": "amplify-stockapp-dev-151619-deployment",
-          "s3Key": "amplify-builds/twitterScraping-37527759426852516354-build.zip",
+          "s3Key": "amplify-builds/twitterScraping-395a4c30485a69633149-build.zip",
           "databasename": "test",
           "limit": "3"
         },
         "newsArticleScrapingAndSentimentAnalysis": {
           "secretsPathAmplifyAppId": "d2l3aeky8jssr5",
           "environment": "prod",
           "databasename": "test",
-          "crontime": "30"
+          "crontime": "30",
```

Comment:
The changes in this file are separate to the user validation? Not sure how this fits in if you could give a small explanation

Reply:
Sorry I'm half asleep here, now I see that this is the same problem from earlier. All good, but we need to figure out what's going on with it for future reference